.github/workflows/bindings.yml

```diff
@@ -45,13 +45,13 @@ jobs:
           fi
       - name: Upload ts bindings artifact on error
         if: failure()
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
```

Comment:
maybe use `v4` instead? https://github.blog/changelog/2024-02-13-deprecation-notice-v1-and-v2-of-the-artifact-actions/

Reply:
udpated!